### DISCUSSION TITLE
Block Feature

### DISCUSSION THREAD

**Discussion by OfshoX** [benawad/dogehouse#199] [Ideas]
I think a block feature will be useful, it will let you block and ignore people who are annoying you.

It can:
- mute them for you
- hide their messages

**Comment by amitojsingh366** [comment]
Seems like a nice idea!

**Reply by vi241** [reply]
it was made already

**Reply by vi241** [reply]
lol

**Reply by vi241** [reply]
it was posted on Feb 14

**Comment by dk-raw** [comment]
There is a blocking-feature now.
